### PR TITLE
Adds cocina support methods.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made? 🤔
Allow the methods to be re-used in DSA and robots.

## How was this change tested? 🤨

Unit

⚡ ⚠ If this change has cross service impact, including data writes to shared file systems, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: lib/sul_orcid_client/contributor_mapper.rb [hunk 1]
```diff
@@ -68,12 +64,12 @@ def map_attributes
     end
 
     MARC_RELATOR_MAP = {
```
Comment (justinlittman) [inline lib/sul_orcid_client/contributor_mapper.rb:66]: This is an unrelated fix that accidentally made it into the PR. The capitalization was wrong on the orcid terms.

File: lib/sul_orcid_client/cocina_support.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  class CocinaSupport
+    # @param [Cocina::Models::Contributor] contributor to check
+    # @return [Boolean] true unless the contributor has a citation status of false
+    def self.cited?(contributor)
+      contributor.note.none? { |note| note.type == "citation status" && note.value == "false" }
+    end
+
+    # @param [Cocina::Models::Contributor] contributor to check
+    # @return [String, nil] orcid id including host if present
+    def self.orcidid(contributor)
```
Comment (lwrubel) [inline lib/sul_orcid_client/cocina_support.rb:12]: Am I understanding correctly that this method and `cited_orcids` don't replace any current code in the client, they're being defined for future use by DSA? Since they don't consider ROR identifiers, I'm assuming there is some work to do in DSA (guessing for pushing the work to ORCID?) that will require specifically checking for ORCID.

**Review by lwrubel — APPROVED** [review]

File: lib/sul_orcid_client/cocina_support.rb (hunk 2 above)
Comment (justinlittman) [reply lib/sul_orcid_client/cocina_support.rb:12]: These will be used in DSA and robots (which is why added to this library).
